src/base/forms/_settings.forms.scss

```
@@ -35,6 +35,7 @@ $input-shadow-focus: $input-shadow !default;
   appearance: none;
   background-color: $input-background;
   box-shadow: $input-shadow;
+  font: inherit;
```

`font-size` already exists below, and the intention of this PR is to reset the family only.
```suggestion
  font-family: inherit;
```